showcase/src/app/demo-ui/pages/dropdown/demo-dropdown-page.component.html

```diff
@@ -88,18 +100,20 @@ <h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
 		filesPath="dropdown/multiple-required-selection"
 		[exampleTitle]="'SHOWCASE.DEMO.DROPDOWN.MULTIPLE_REQUIRED' | translate"
 	>
-		<stark-dropdown
-			dropdownId="multipleServicesRequiredDropdown"
-			[options]="serviceDropdownOptions"
-			[value]="selectedRequiredServices"
-			optionIdProperty="id"
-			optionLabelProperty="value"
-			placeholder="SHOWCASE.DEMO.DROPDOWN.SELECT_ONE_SERVICE"
-			(selectionChanged)="multipleServicesRequiredDropdownOnChange($event)"
-			multiSelect
-			required
-		>
-		</stark-dropdown>
+		<mat-form-field>
+			<stark-dropdown
+				dropdownId="multipleServicesRequiredDropdown"
+				[options]="serviceDropdownOptions"
+				[value]="selectedRequiredServices"
+				optionIdProperty="id"
+				optionLabelProperty="value"
+				[placeholder]="'SHOWCASE.DEMO.DROPDOWN.SELECT_ONE_SERVICE' | translate"
+				(selectionChanged)="multipleServicesRequiredDropdownOnChange($event)"
+				multiSelect
+				required
```

Comment:
Same issue here: it is not working as expected... although it is required, the dropdown is not marked in red when there is no value selected....